pipeline/bedstat.py

```diff
@@ -27,6 +29,10 @@
     '--open-signal-matrix', type=str, required=False, default=None,
     help='a full path to the openSignalMatrix required for the tissue '
          'specificity plots')
+parser.add_argument(
```

Comment:
what if the path is not provided? It seems like it is required for the command in 102. 

I propose we keep it optional but fall back to refgenie. We can use `RefGenConf.seek` to get the path in case the `$REFGENIE` env var is set and fail otherwise.

Reply:
this might be outdated, but useful: https://github.com/databio/bedmaker/blob/71961516a95a35e191b3c13bbe5b7fb904fa6337/bedmaker.py#L97-L119